assets/css/faq.css

```diff
@@ -16,3 +16,74 @@
 .faq-item p {
 margin-bottom: 2rem;
 }
+
+/* ----- faq-description  section ----- */
+.faq-description {
+    background: linear-gradient(0deg, var(--dark) 20%, #070302 100%) no-repeat;
+    padding: 40px 0px 0px;
+    text-align: center;
+}
+.faq-description .divider {
+    margin: 0px;
+    position: relative;
+    bottom: -10px;
+}
+.faq-hero {
+    background-size: cover; 
+    background-position: center;
+    background-color: #020202;
+}
+@media screen and (max-width:1257px) {
+    .faq-hero {
+        background-size: contain;background-size: auto;background-color: #020202;
```

Comment:
This style has two background sizes. Only the last one will function.
background-size: contain;
background-size: auto;